### PR TITLE
Implement ternary operator fixes in VB and add IIF -> If analyzer/fix

### PR DESCRIPTION
**Pull request by jwooley**
Translates ternary code fixes to VB and adds a new code fix to translate the old IIF to If for VB.

### CODE REVIEW

**Comment by giggio** [comment]
@jwooley Some tests are failing, can you check?

**Comment by jwooley** [comment]
I'll recheck this evening. Was passing locally.

**Comment by jwooley** [comment]
I tried updating the project to the current master, but am now getting an error in the XUnit runner with the xUnit RC 2 update in that build. I ran out of time to debug further tonight. I'll try to look at it again tomorrow. I did see that there's already a xUnit RC3 update in nuget as well, so I wonder if it would be an issue to try bumping that up on more rev.

**Comment by giggio** [comment]
@jwooley Please update to RC1 using current HEAD.

**Comment by jwooley** [comment]
Build failing on C# tests which were not impacted by this pull request.

6C:\projects\code-cracker\packages\xunit.runners.2.0.0-beta5-build2785\tools\xunit.console.x86.exe "C:\projects\code-cracker\test\CSharp\CodeCracker.Test\bin\Debug\CodeCracker.Test.CSharp.dll" -appveyor

**Comment by giggio** [comment]
@jwooley I rebuilt it, and it ok as you can see on the appveyor status.

File: src/Common/CodeCracker.Common/DiagnosticId.cs [hunk 1]
```diff
@@ -57,6 +57,7 @@ public enum DiagnosticId
         UseConfigureAwaitFalse = 70,
         IntroduceFieldFromConstructor = 71,
         RemoveAsyncFromMethod = 72,
-        AddBracesToSwitchSections = 73
+        AddBracesToSwitchSections = 73,
+        TernaryOperator_Iif = 74
```
Comment (giggio) [inline src/Common/CodeCracker.Common/DiagnosticId.cs]: `74` is taken with another analyzer (see the [diagnostic ids](https://github.com/code-cracker/code-cracker/wiki/DiagnosticIds) page on the wiki). You may grab the next number and update the @jwooley!

File: src/VisualBasic/CodeCracker/CodeCracker.vbproj [hunk 2]
```diff
@@ -151,7 +153,6 @@
   </ItemGroup>
   <ItemGroup>
     <Analyzer Include="..\..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.0.0-rc1\tools\analyzers\Microsoft.CodeAnalysis.Analyzers.dll" />
-    <Analyzer Include="..\..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.0.0-rc1\tools\analyzers\VB.NET\Microsoft.CodeAnalysis.VisualBasic.Analyzers.dll" />
```
Comment (giggio) [inline src/VisualBasic/CodeCracker/CodeCracker.vbproj]: Probably causing the same problem as the C# one, right? Failing builds?

File: src/VisualBasic/CodeCracker/My Project/AssemblyInfo.vb [hunk 3]
```diff
@@ -12,7 +12,7 @@ Imports System.Runtime.InteropServices
 <Assembly: AssemblyTrademark("")>
 <Assembly: AssemblyCulture("")>
 <Assembly: ComVisible(False)>
-<Assembly: NeutralResourcesLanguage("en")>
+<Assembly: NeutralResourcesLanguage("en-us")>
```
Comment (giggio) [inline src/VisualBasic/CodeCracker/My Project/AssemblyInfo.vb]: Why this change? Did you do it, or did VS did it?

File: src/VisualBasic/CodeCracker/CodeCracker.vbproj (hunk 2 above)
Comment (jwooley) [inline src/VisualBasic/CodeCracker/CodeCracker.vbproj]: Yes, this is why I had to remove the analyzer.
